contracts/library/PoolLib.sol

```diff
@@ -133,7 +133,7 @@ library PoolLib {
         uint256 preBurnBptBal            = bPool.balanceOf(address(this));
 
         // Burn enough BPTs for liquidityAsset to cover defaultSuffered
-        bPool.exitswapExternAmountOut(
+        bPool.exitSwapExternAmountOut(
```

Comment:
This actually needs to stay as is: https://docs.balancer.finance/api/api#exitswapexternamountout

Reply:
Thats why CI is failing

Reply:
Wow, balancer. How could you.